src/app/utils/url-generator.js

```diff
@@ -16,13 +16,16 @@ export const host = () => {
 export const protocolAndHost = () => `${location.protocol}//${host()}`;
 
 export const newSandboxUrl = () => `/s/new`;
```

Comment:
Should this be `/react` now instead of `/new` ? Or is there some backwards compatibility that could break by changing it?

Reply:
It would break the links which people have been sharing, but I agree with you.
May be we can add support for `/react` and keep `/new` for backwards compatibility.

Reply:
Yep, we could add a redirect (in the pages component) from new to React. Let's do this in a separate PR to keep things separate.